src/core/ItemData.h

```diff
@@ -210,6 +210,7 @@ class CItemData : public CItem
   void UpdatePassword(const StringX &password); // use when password changed!
   void SetTwoFactorKey(const StringX& value) { CItem::SetField(TWOFACTORKEY, value); }
   bool SetTotpConfig(const StringX& value) { return SetFieldAsByte(TOTPCONFIG, value.c_str()); }
+  void SetTotpStartTime(time_t t) { CItem::SetTime(TOTPSTARTTIME, t); }
```

Comment:
Is this meant to overload the existing SetTotpStartTime? If so, shouldn’t it also return a bool? Also, can we omit the CItem prefix for consistency with the other method?

Reply:
Yes, this overloads SetTotpStartTime. I copied the interface used by the other fields defined in the same file:

```
  void SetATime(time_t t) {CItem::SetTime(ATIME, t);}  // V30
  void SetCTime(time_t t) {CItem::SetTime(CTIME, t);}  // V30
  void SetXTime(time_t t) {CItem::SetTime(XTIME, t);}  // V30
  void SetPMTime(time_t t) {CItem::SetTime(PMTIME, t);}  // V30
  void SetRMTime(time_t t) {CItem::SetTime(RMTIME, t);}  // V30
```